tests/baselines/reference/objectSpread.types

```diff
@@ -407,4 +407,11 @@ let shortCutted: { a: number, b: string } = { ...o, a }
 >o : { a: number; b: string; }
 >a : number
 
+// non primitive
+let spreadNonPrimitve = { ...<object>{}}
+>spreadNonPrimitve : { constructor: Function; toString(): string; toLocaleString(): string; valueOf(): Object; hasOwnProperty(v: string): boolean; isPrototypeOf(v: Object): boolean; propertyIsEnumerable(v: string): boolean; }

```

Comment:
No. Just like spreading `number` in javascript results in `{}` not `{ toString(), toFixed() ... }`, spreading `object` should not get the properties of `globalObjectType`.

(Note that spreading number in typescript is illegal because it's useless, probably a mistake. Spreading object seems more useful.)